### DISCUSSION TITLE
Add theme music as preinstalled maps

### DISCUSSION THREAD

**Discussion by Givikap120** [ppy/osu#28149] [Ideas] [answered]
Add nekodex - circles! as a map coming with osu-stable preinstall
Same for cYsmix - triangles but for osu-lazer
It feels kinda weird that you have a menu music but you don't have a map for it immediately

**Comment by peppy** [comment, marked as answer]
It comes as a bundled beatmap, which is installed during the setup wizard.

**Reply by Givikap120** [reply]
oh, it was updated not so long ago, sorry
